flow360/component/simulation/web/asset_base.py

```diff
@@ -159,6 +162,39 @@ def from_file(
             length_unit=length_unit,
         )
 
+    @classmethod
+    # pylint: disable=too-many-arguments
+    def from_local_storage(cls, id: str = None, local_storage_path=""):
```

Comment:
Interesting that pylint did not complain here. But still can you change this to asset_id?